src/app.rs

```diff
@@ -623,6 +746,30 @@ impl eframe::App for RustyAutoClickerApp {
                         );
                     });
                 });
+                ui.separator();
+                ui.label("Movement delay");
+
+                ui.with_layout(egui::Layout::right_to_left(egui::Align::TOP), |ui| {
+                    ui.label("ms");
+                    if self.is_autoclicking || self.hotkey_window_open {
+                        ui.set_enabled(false);
+                    };
+                    ui.add(
+                        egui::TextEdit::singleline(&mut self.movement_ms_str)
+                            .desired_width(40.0f32)
+                            .hint_text("20"),
+                    );
+
+                    ui.label("sec");
+                    if self.is_autoclicking || self.hotkey_window_open {
+                        ui.set_enabled(false);
+                    };
+                    ui.add(
+                        egui::TextEdit::singleline(&mut self.movement_sec_str)
+                            .desired_width(40.0f32)
+                            .hint_text("0"),
+                    );
+                });
```

Comment:
Wrap in a 
```rust
ui.horizontal_wrapped(|ui| {
... 
}
```
please, this will make the label and input fields on the same lines

----------------

Also add "(Humanlike only)" to the "Movement delay" label please.